launchdarkly/README.md

```diff
@@ -1,45 +1,56 @@
 # LaunchDarkly Integration
 
-## Overview
+### Overview
 
-This integration tracks metrics from [LaunchDarkly's relay proxy][1], such as number of proxied stream connections and proxied API route requests.
+Deploying features with LaunchDarkly is great, but it can be difficult to see
+the effects of these deployments on users' services or systems. Datadog's
+monitoring and analytics tools for modern DevOps teams allow their customers to
+observe the health of their systems over time. The LaunchDarkly integration with
+Datadog exposes LaunchDarkly events in context with events gathered by the
+Datadog agents. This makes it easier to reason about changes to customers'
+systems as a result of feature deployments. For example, if a feature gets
+deployed that causes a service to slow down, a DevOps engineer can see the cause
+within Datadog.
```

Comment:
```suggestion
The Datadog-LaunchDarkly integration allows you to see the effects of your LaunchDarkly feature deployments on your customer's services or systems. For instance, if a deployed feature causes a service to slow down, you would be able to see the cause within Datadog.
This integration displays LaunchDarkly events in the [Datadog Event Stream](https://app.datadoghq.com/event/stream) and collect metrics from [LaunchDarkly's relay proxy](https://docs.launchdarkly.com/docs/the-relay-proxy), such as the number of proxied stream connections and proxied API route requests.
```

There is no need to re-explain what LaunchDarkly and Datadog are and do here, only what's the end-goal of this integration.
From a style perspective there is no need to break lines to respect a max number of characters. It's a doc not a piece of code :)